### PR TITLE
Add noscript tag

### LINKED ISSUE #3759: Pages with the Scratch 3.0 Theme are blank if JavaScript is disabled
If you attempt to view any pages which have the Scratch 3.0 theme (the home page, project page, etc.) while JavaScript is disabled, the page is blank without any explanation. There should be some kind of message which informs the user that they need to enable JavaScript to view the page.

### Steps to Reproduce

1. Block JavaScript for scratch.mit.edu in your browser settings
2. Visit a page which has the 3.0 theme (homepage, project page, project editor, etc.)

### Operating System and Browser

My browser / operating system: ChromeOS 12739.94.0, Chrome 80.0.3987.137, No Flash version detected

### PR DESCRIPTION
**Pull request by apple502j**
### Resolves:
Resolves #3759 

### Changes:
Adds "JavaScript is required to display this page. Please enable JavaScript in order to continue." message.

### CODE REVIEW

File: src/template.ejs [hunk 1]
```diff
@@ -53,6 +53,9 @@
     </head>
 
     <body>
+        <noscript>
+            <p>JavaScript is required to display this page. Please enable JavaScript in order to continue.</p>
+        </noscript>
```
Comment (GrahamSH-LLK) [inline src/template.ejs:58]: Looks good! I think we would have to style it, though. Is it styled?

**Comment by Accio1** [comment]
Shouldn't this be stylized using CSS?

**Comment by GrahamSH-LLK** [comment]
> Shouldn't this be stylized using CSS?

Ha! We both said the same thing. I think @benjiwheeler said to give him basic outline, and the ST would style it.

**Comment by benjiwheeler** [comment]
Thanks. The next step is for me to take this to the Design team and get their feedback.

**Comment by BoomerScratch** [comment]
@R4356th Maybe tell the user this?
> JavaScript is required to display this page. Please [enable JavaScript](https://enablejavascript.io) in order to continue. If your browser does not support JavaScript, read about common workarounds.

(link needed for the "read about common workarounds")
We sadly can't add a link to the offline editor because all download pages require JS!

**Comment by R4356th** [comment]
No, @BoomerScratch. That is the download link for 2.0. I am talking about these- https://downloads.scratch.mit.edu/desktop/Scratch%20Desktop%20Setup%203.11.1.exe, https://downloads.scratch.mit.edu/desktop/Scratch%20Desktop-3.11.1.dmg for Windows and MacOS respectively. Now, as for ChromeOS and Android, there should be links to the Google Play Store. All of them cannot stay there as that will take up space. So, I think there should be an only-HTML-and-CSS page containing these links.

**Comment by GrahamSH-LLK** [comment]
@R4356th What if we made a non-react generated copy of [scratch.mit.edu/download](https://scratch.mit.edu/download)? We would have to do some fancy html, but we could probably get it to work.

**Comment by BoomerScratch** [comment]
@R4356th Yes, but these links have to be updated over time. The link on maintenance mode also takes you to the 2.0 download:

I agree about the only-HTML-and-CSS one.

**Comment by benjiwheeler** [comment]
FWIW, I think because this is at least an improvement over what we currently have, we should be able to use it, and then come back to doing a more complete design.

**Comment by benjiwheeler** [comment]
blocked on discussing with design team

**Comment by benjiwheeler** [comment]
Spoke with design team and agreed that this is an improvement! Per their feedback I changed the wording slightly.

Thanks, @apple502j!

**Pull request merged**